waiter/src/waiter/metrics.clj

```diff
@@ -517,3 +517,16 @@
    (.update timer
             (t/in-millis duration)
             TimeUnit/MILLISECONDS)))
+
+(defmacro endpoint-with-waiter-metrics
+  "Calls body, wrapping with timer, count, concurrent count, and rate metrics"
+  [classifier nested-path & body]
+  `(:out (with-timer
+           (waiter-timer ~classifier ~@nested-path) ; timer has both timer and rate
+           (do
+             (counters/inc! (waiter-counter ~classifier ~@nested-path))
```

Comment:
I think this is the first we are introducing a timer and counter with the same name. Should they have different names, e.g. `total` for the counter?

Reply:
they already have "timer" or "counter". Don't see what info is added